scss/components/_control-bar.scss

```diff
@@ -81,6 +81,13 @@
 
     .oo-control-bar-item:focus .oo-icon {
       @include highlight-base();
+      @include highlight-tab();
+      opacity: 1 !important;
+    }
+
+    .oo-control-bar-item:hover .oo-icon {
+      @include highlight-base();
+      @include remove-highlight-tab();
```

Comment:
Can we keep the focus styles when hovering? I think it's more accurate since hovering doesn't remove the focused state.